### DISCUSSION TITLE
dropdown show error

### DISCUSSION THREAD

**Discussion by paingchan** [awcodes/filament-quick-create#46] [Q&A] [answered]
https://github.com/awcodes/filament-quick-create/assets/54994337/b3cb61db-c0ad-4cb2-9263-b5307379cd23

<img width="707" alt="image" src="https://github.com/awcodes/filament-quick-create/assets/54994337/13bfc034-cda0-4f8a-9da9-de396e05cc3b">
<img width="707" alt="image" src="https://github.com/awcodes/filament-quick-create/assets/54994337/944bb79b-2cfa-4799-9dc4-28377b764737">

**Comment by awcodes** [comment]
Does this happen without slideover()?

Haven't had any other reports of the issue.

**Reply by paingchan** [reply]
Now , In Chrome working Only in Safari

**Comment by awcodes** [comment, marked as answer]
This is fixed with latest version. Cheers.

**Reply by paingchan** [reply]
Thank u It Working Now .
